extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -52,6 +52,12 @@ amp-embed iframe {
 
 amp-ad[sticky] {
   visibility: hidden;
+  align-items: center;
+}
+
+amp-ad[amp-story] iframe {
```

Comment:
this should probably be in story ad css?

Reply:
moved.